.github/workflows/cron-community-issue.yml

```diff
@@ -0,0 +1,23 @@
+name: Create an issue for tracking meeting
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 18 * * WED'
```

Comment:
does this mean at 6pm GMT Wednesday?

Reply:
6 PM UTC....which should be 1 PM ETC unless my math is trash